package.json

```diff
@@ -75,8 +72,8 @@
   "scripts": {
     "all": "npm run lint && npm run test && npm run accept",
     "babel-watch": "babel src -d dist --watch --source-maps --copy-files",
-    "coverage": "istanbul cover ./node_modules/mocha/bin/_mocha -- -R spec test/ --compilers js:babel-register",
-    "coveralls": "istanbul cover ./node_modules/mocha/bin/_mocha --report lcovonly -- -R spec --compilers js:babel-register test/ && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage",
+    "coverage": "istanbul cover ./node_modules/.bin/_mocha test/** -- -R spec --compilers js:babel-register",
+    "coveralls": "istanbul cover ./node_modules/.bin/_mocha --report lcovonly -- -R spec --compilers js:babel-register test/** && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage",
```

Comment:
do you know why we're not using the `mocha` command instead ([example](https://github.com/particle-iot/config/blob/main/package.json#L21))?

Reply:
Despite `mocha` being a wrapper for `_mocha` (see https://github.com/gotwarlost/istanbul/issues/436#issuecomment-139461731), it fails returning a `No coverage information was collected, exit without writing coverage information` error. What I could do is execute `_mocha` instead of `./node_modules/.bin/_mocha`.